packages/cli/src/external-secrets.ee/providers/vault.ts

```diff
@@ -419,7 +418,7 @@ export class VaultProvider extends SecretsProvider {
 		listPath += path;
 		let listResp: AxiosResponse<VaultResponse<VaultSecretList>>;
 		try {
-			const shouldPreferGet = preferGet();
+			const shouldPreferGet = Container.get(ExternalSecretsConfig).preferGet;
```

Comment:
We can't use DI here?

Reply:
We can. but that broke the tests. and refactoring those tests got very messy. I will clean this up in a subsequent PR.